### DISCUSSION TITLE
Any way to use a player other than mpv and vlc?

### DISCUSSION THREAD

**Discussion by haha-lolcat** [mov-cli/mov-cli#334] [Q&A] [answered]
Hi all,

I was wondering if there was a way to use a different app for mov-cli for streaming.
Specifically, I would like to use the native windows app "Media Player" because it's the only app that allows me to stream from my pc to my tv. I can't do it from VLC because my tv doesn't have chromecast, and mpv just doesn't have media sharing whatsoever.
Any ideas for this? Or do we need a new plugin for mov-cli?

**Comment by THEGOLDENPRO** [comment, marked as answer]
Yeah, you can use any player with mov-cli that is available from your system's path and can be called with the media's URL as another parameter like so:

```sh
player.exe {url}
```

Looking into the wiki for the config's `player` key you can see exactly that is possible: https://github.com/mov-cli/mov-cli/wiki/Configuration#player

I don't have a windows system with me at the moment so I can't figure out or test it but *maybe add `wmplayer` to the player key*. You'll have to figure it out yourself but this might help: https://learn.microsoft.com/en-us/windows/win32/wmp/command-line-parameters

```toml
player = "wmplayer"
```
